backend/apps/apiboss/lib/apiregistry_extensions/apibossrateenforcer.js

```diff
@@ -41,7 +41,7 @@ function checkSecurity(apiregentry, url, req, headers, servObject, reason) {
     const rateLimits = CLUSTER_MEMORY.get(RATELIMIT_DISTM_KEY)[key]; 
     if (!rateLimits) return true; // no SLA for this API Key
 
-    const apiCallMap = _getAPICallMap(); if (!apiCallMap[key]) apiCallMap[key] = {...CALL_MAP_TEMPLATE};
+    const apiCallMap = _getAPICallMap(); if (!apiCallMap[key]) apiCallMap[key] = JSON.parse(JSON.stringify(CALL_MAP_TEMPLATE));
```

Comment:
Utils have a deep clone function. Please use that. So that if we change the deep cloning strategy in the future we don't need to comb entire code everywhere.

Reply:
Ok Rohit , sending again with utils clone function